docs/topics/content_components.md

```diff
@@ -160,7 +160,7 @@ components = [
 
 Attachment media only allows you to use the `attachment://<file_name>` protocol.
 
-You can additional mark attachments as spoilered.
+You can mark attachments as spoilers.
```

Comment:
Saying something nice:
Good catch, the old did not make any sense.

Reply:
> Saying something nice: Good catch, the old did not make any sense.

Almost missed it :D